File: programmers/09. 서울에서 김서방 찾기.js
function solution(seoul) {
    // 인덱스 위치를 찾음
    for(let i=0; i<seoul.length; i++) {
        // console.log(seoul[i])
        if(seoul[i] === 'Kim') {
           return `김서방은 ${i}에 있다`;
        }
    }
}

 // string 형 배열 seoul
console.log(solution(['Jane', 'Kim']))
console.log(solution(['Kim','Jane']))
console.log(solution(['Jane','Dev', 'Hi','Kim']))

File: programmers/java/09. 서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String answer = "";
        
        for(int i=0; i<seoul.length; i++) {
            if(seoul[i].equals("Kim")) {
                answer = "김서방은 " + i + "에 있다";
                break;
            }
        }
        return answer;
    }
}


//  int x = Arrays.asList(seoul).indexOf("Kim");
//  return "김서방은 "+ x + "에 있다";